### PR TITLE
Dashboard skeleton

### PR DESCRIPTION
**Pull request by RetiFier**
### What this PR does (required):
- Dashboard Skeleton: An Update and Redesign for nav

### Screenshots / Videos (front-end only):
image
image

### Any information needed to test this feature (required):
- With and without authentication, check av bar.

### CODE REVIEW

File: client/src/components/GustMenu/GustMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const GuestMenu = ({}: Props): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+  return (
+    <Box justifyContent={'center'}>
+      <Link to={'/'} className={classes.linkBtn}>
+        <Typography display={'inline'} fontWeight={800} fontSize={11}>
+          BECOME A SITTER
+        </Typography>
```
Comment (moffatethan) [inline client/src/components/GustMenu/GustMenu.tsx]: MUI v5 has a `sx` property that is generally preferred now. Example below. 

```js
<Typography sx={{ display: 'inline', fontWeight: 800, fontSize: 11 }}>
    BECOME A SITTER
</Typography>
```

**Review by moffatethan — CHANGES_REQUESTED** [review]
Overall looks decent, I would've like to see the navbar match the UI a bit better. Also, is your branch based properly? I am seeing back-end work here.

File: client/src/components/GustMenu/GustMenu.tsx (hunk 1 above)
Comment (RetiFier) [reply client/src/components/GustMenu/GustMenu.tsx]: I think I make little mistakes when branching. I am working on the backend also When the backend is reviewing, I created branch from the backend branch.

**Review by moffatethan — APPROVED** [review]
Good to go

**Pull request merged**